docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '2'
```

Comment:
The last version of the compose file format is 3.8. 
Why do you use the version 2 ?
https://docs.docker.com/compose/compose-file/

Reply:
For exemple, I have tested my file as version 3.3 on Solus, it's still working as expected. But docker-compose 3.8 is still not supported on Solus (while the docker core is obtained through the docker repo, docker-compose is obtained through the OS repos, which tend to be not recent enough on some distributions).

Why not keep it a low number, and benefit from docker-compose backward compatibility to make it work on as much platforms as possible? Otherwise, I would still tend to go for a non-latest version, like the 3.3 that seems to work perfectly on all my setups.

@vamonte